apps/examples/lcd_test/example_lcd.c

```diff
@@ -371,6 +425,11 @@ int lcd_test_main(int argc, char *argv[])
 	int fd = 0;
 	int p = 0;
 	char port[20] = { '\0' };
+	pid_t touch = task_create("touch", SCHED_PRIORITY_DEFAULT, 8096, touch_test, NULL);
```

Comment:
This is a LCD test example. I mean the main purpose is testing LCD. But some LCD can be touchless. Then, is this example ok to run? No need to cover new codes with touch config?